Jenkinsfile

```diff
@@ -40,21 +40,21 @@ pipeline {
           }
           steps {
             script {
-              env.DOCKER_TAG = 'openpaas-branch-master'
+              env.DOCKER_TAG = 'branch-master'
```

Comment:
Opened to suggestions here :
* branch-master ?
* latest ?
* ...?

Reply:
branch-master :+1: